src/x11/tray_manager.cpp

```diff
@@ -67,25 +67,26 @@ tray_manager::~tray_manager() {
   deactivate();
 }
 
-void tray_manager::setup() {
+void tray_manager::setup(const string& tray_module_name) {
   const config& conf = config::make();
   auto bs = conf.section();
-  string position;
+  string position = conf.get(bs, "tray-position", string("none"));
```

Comment:
```suggestion
  string position = conf.get(bs, "tray-position", "none"s);
```

We use `""s` for string literals